CHANGELOG.md

```diff
@@ -1,6 +1,54 @@
-## 3.0.0 (Unreleased)
+## 3.0.0 (October 16, 2020)
+
+FEATURES
+
+* **New Resource**: `vcd_vapp_access_control` Access control for vApps [#543](https://github.com/vmware/terraform-provider-vcd/pull/543)
+* **New Data Source**: `vcd_org_user` Org User [#543](https://github.com/vmware/terraform-provider-vcd/pull/543)
+* **New Resource**: `vcd_vm_sizing_policy` VDC VM sizing policy [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* **New Data Source**: `vcd_vm_sizing_policy` VDC VM sizing policy [#553](https://github.com/vmware/terraform-provider-vcd/pull/553)
+* **New Resource**: `vcd_edgegateway_settings` Changes LB and FW global settings for Edge Gateway [#557](https://github.com/vmware/terraform-provider-vcd/pull/557)
+* **New Resource**: `vcd_external_network_v2` with NSX-T support [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_external_network_v2` with NSX-T support [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_vcenter` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_portgroup` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_nsxt_manager` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
+* **New Data Source**: `vcd_nsxt_tier0_router` [#560](https://github.com/vmware/terraform-provider-vcd/pull/560)
```

Comment:
Would be good to have a few words where they are used here in the changelog.
CC: @Didainius

Reply:
All of these datasources are useful for `vcd_external_network_v2` to specify 

`vcd_vcenter` and `vcd_portgroup` are used to create NSX-V backed `vcd_external_network_v2`
`vcd_nsxt_manager` and `vcd_nsxt_tier0_router`  are used to create NSX-T backed `vcd_external_network_v2`